code/game/objects/structures/cliff.dm

```diff
@@ -201,56 +216,121 @@ two tiles on initialization, and which way a cliff is facing may change during m
 			var/mob/living/carbon/human/H = L

```

Comment:
Can we invert the `if(istype(T))` check above this into an early return to dedent the rest of this function?